diff --git a/lib/github_changelog_generator/generator/section.rb b/lib/github_changelog_generator/generator/section.rb
index 538b2b1..a9be683 100644
--- a/lib/github_changelog_generator/generator/section.rb
+++ b/lib/github_changelog_generator/generator/section.rb
@@ -81,7 +81,7 @@ module GitHubChangelogGenerator
       if @options[:usernames_as_github_logins]
         "#{line} (@#{user['login']})"
       else
-        "#{line} ([#{user['login']}](#{user['html_url']}))"
+        "#{line} ([@#{user['login']}](#{user['html_url']}))"
       end
     end
 
